src/wp-includes/script-loader.php

```diff
@@ -106,7 +107,8 @@ function wp_default_packages_vendor( $scripts ) {
 		'wp-polyfill-dom-rect'        => '3.104.0',
 		'wp-polyfill-element-closest' => '2.0.2',
 		'wp-polyfill-object-fit'      => '2.3.5',
-		'wp-polyfill'                 => '7.4.4',
+		'wp-polyfill-build'           => '7.4.4',
```

Comment:
I think I forgot to updat the version here:
```suggestion
		'wp-polyfill-build'           => '6.2.0',
```